.gitignore

```diff
@@ -5,3 +5,4 @@
 uuagc/trunk/dist/*
 dist-newstyle/
 cabal.project.local
+cabal.project.local~
```

Comment:
This looks like an emacs backup file, probably better to add `*~` here

Reply:
I think it is actually generated by running `cabal configure` (I use vim myself). If the `cabal.project.local` file already exists, it saves the old file with that suffix.

Reply:
The "official" haskell gitignore also has `cabal.project.local~` listed: https://github.com/github/gitignore/blob/master/Haskell.gitignore. So I think leaving it like this is fine.